package.json

```diff
@@ -25,6 +25,11 @@
     "url": "https://github.com/OMGDuke/SDH-GameThemeMusic/issues"
   },
   "homepage": "https://github.com/OMGDuke/SDH-GameThemeMusic#readme",
+  "remote_binary": {
```

Comment:
This should be an array of remote binary objects

Reply:
Ah, I think I got confused by the singular in the name and by [the documentation](https://wiki.deckbrew.xyz/en/plugin-dev/getting-started#package-metadata)—this also explains why it didn't work before. Thanks for pointing it out, the binary is now correctly retrieved!